tox.ini

```diff
@@ -61,26 +61,31 @@ extras = tests
 commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
+[testenv:py312]
+extras = tests
+commands =
+    pytest --cov buku -vv -m "not non_tox" {posargs}
+
 [testenv:quick]
-basepython = {env:BASEPYTHON:py311}
+basepython = {env:BASEPYTHON:py312}
```

Comment:
Note that this change replaces the _default_ python version (unless explicitly specified)

Reply:
but I guess we can find out the issues when we make this switch?